plugins/env/app/views/samson_env/_project_form.html.erb

```diff
@@ -15,11 +15,11 @@
     %>
   <% end %>
 
-  <% if defined?(EnvironmentVariable) && folder = EnvironmentVariable.config_service_folder(@project, required: false) %>
+  <% if defined?(EnvironmentVariable) && folder = EnvironmentVariable.config_service_label(@project) %>
```

Comment:
_label -> folder + path -> folder for consistency ?

Reply:
Clarification?

Reply:
rename label to folder ... path/label/folder/key/bucket is too many words :D